src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -167,11 +160,16 @@ public function setupFieldAssociations($fields, $table, $fieldConditions, $strat
             ]);
         }
 
+        $conditions = ["$targetAlias.model" => $model];
+        if (!$this->_config['allowEmptyTranslations']) {
+            $conditions["$targetAlias.content !="] = '';
+        }
```

Comment:
The not equal operator is either syntax, unless you're using [access or db2](http://stackoverflow.com/a/723426) in which case - you probably have 98 more problems.

Would you mind demonstrating "a difference", pointing at a reference?